### DISCUSSION TITLE
Add "involves:" to shortcuts

### DISCUSSION THREAD

**Discussion by TPS** [community/community#11767] [Mobile] [answered]
I'd like the `involves:` keyword (which combines multiple of the others) added to "Shortcuts."  It's what I use most, especially as `involves:TPS`, which works just fine in mobile search. The `Recent` section @ bottom of logged-in homepage is good for just a few issues, but needs to be selectable to expand into more, & include all other activity types.

**Comment by masachs** [comment]
Hey @TPS, thanks for reaching out!

I'll add this to our list of things to explore for the future. In the meantime, you should be able to achieve this via a workaround:

* Go to the list for the type of content your filtering for (i.e. Issues, Pull Requests, etc. either on the home view or on a repository)
* Manually type in your term into the search box (i.e. `involves:TPS` going by your example above)
* Hit return and watch this term then move down into the filter list.
* Hit the button on the left of the filter list and create a shortcut from there. The resulting shortcut will then include this custom filter as well.

A bit round-about, I know, but I hope this will unblock you in the meantime!

**Reply by TPS** [reply]
@masachs That's not bad, but seems like it'd take up too much time.  Just a general `involves:TPS` would give me great overview (just like it does in the search box), then I could drill down when I see what I want.

**Comment by Mahir6** [comment]
- _[]()_~~
~~__

**Comment by TigersWay** [comment]
> 
>     * Hit return and watch this term then move down into the filter list.
> 
Not sure about today (more than 1 year after) but it doesn't seem possible anymore... And still needed somehow!

**Reply by TPS** [reply]
It still works.…

**Reply by TigersWay** [reply]
Great then! But I did not understand the procedure

**Comment by TPS** [comment, marked as answer]
This is now implemented in the app's Issues (`Involved`), PRs (`Involved`), & Discussions (`Commented by me`) links, & is fantastic because they're remembered as a default.  Thanks very much, @GitHub. 🙇🏾‍♂️
